common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -109,6 +109,7 @@ function (Iterator) {
             speedsContainer.html(speedsList.join(''));
             this.speedLinks = new Iterator(speedsContainer.find('.speed-option'));
             this.state.el.find('.secondary-controls').prepend(this.el);
+            this.setActiveSpeed("1.0"); // default state on initial render
```

Comment:
@clrux In attached screenshot the speed is `1.50x`  but it's highlighting `1.0`. Should't it highlight the given speed value ?